### ISSUE TITLE
Install script does not work

### ISSUE THREAD

**Issue by HGustavs** [HGustavs/LenaSYS#5641]
It would be great if we could collect the reasons why the install script failed in this issue so that we can fix it if there is time left at some point

**Comment by a17adajo** [comment]
![image](https://user-images.githubusercontent.com/49197690/55545847-66d50900-56ce-11e9-8be8-d6eb8e365dca.png)


![image](https://user-images.githubusercontent.com/49197690/55545890-7d7b6000-56ce-11e9-805e-3ec47db4f4cf.png)

**Comment by HGustavs** [comment]
Im thinking that removing the offending line from the testdata for user_course ... unclear what is wrong with "BoLindberg"

groupkind seems to be a column that was added efter the install script was made....

Im thinking that this column needs to be added to the install script.

~~~
CREATE TABLE groups (
  groupID INTEGER unsigned NOT NULL AUTO_INCREMENT,
  groupKind VARCHAR(16) NOT NULL,
	groupVal VARCHAR(8) NOT NULL,
	groupInt INTEGER NOT NULL,
  PRIMARY KEY (groupID)
) CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE=InnoDB;
~~~
seems like that code was not in the install script SQL....

Im thinking that the better SQL is in
https://github.com/HGustavs/LenaSYS/blob/master/Shared/SQL/init_db.sql

**Comment by HGustavs** [comment]
It is not good that we have two un-synched sql files that contain "mostly" the same sql.

IMHO The install script should be using the Shared/SQL/init_db.sql ???

**Comment by a17adajo** [comment]
Fixed in #5908
